docs/reference/commandline/service_create.md

```diff
@@ -320,9 +320,20 @@ volumes in a service:
       </ul></p>
     </td>
   </tr>
+  <tr>
+    <td><b>consistency</b></td>
+    <td></td>
+    <td>
+      <p>The consistency requirements for the mount:
+         <tt>default</tt>,
```

Comment:
Ordering again, and questions about `default`. Also, this is a great time to explain what each of them means. I'd put them into a `<ul>`.